code-challenges.test.js

```diff
@@ -52,5 +121,31 @@ const numbersToAdd2 = [0, 7, -8, 12]
 const numbersToAdd3 = []
 // Expected output: []
 
+describe("addItUp", () => {
+
+    it("take numbersToAdd1 and pass the values and return array of the accumulating sum", () => {
+
+        expect(addItUp()).toEqual([2, 4, 45, 9])
+
+    })
+})
+
```

Comment:
Looks like you got the input and expected output mixed up here. You didn't pass anything into addItUp(). The test will pass if you put one of the variables in the addItUp() function and then set it .toEqual(the expected output).  Good work, just a little confusion.